go/vt/sqlparser/rewriter.go

```diff
@@ -0,0 +1,1320 @@
+// Code generated by visitorgen/main/main.go. DO NOT EDIT.
+
+package sqlparser
+
+//go:generate make visitor
+
+import (
+	"reflect"
+)
+
+type replacerFunc func(newNode SQLNode, parent SQLNode)
+
+// application carries all the shared data so we can pass it around cheaply.
+type application struct {
+	pre, post ApplyFunc
+	cursor    Cursor
+}
+
+func replaceAliasedExprAs(new SQLNode, parent SQLNode) {
```

Comment:
Could you use something other than `new`? It could confuse people that use it as keyword.